mycroft/skills/common_play_skill.py

```diff
@@ -19,6 +19,7 @@
 from .mycroft_skill import MycroftSkill
 from .audioservice import AudioService
 
+from mycroft.util.log import LOG
```

Comment:
this should be placed above the local relative imports

Reply:
done